.envrc.example

```
@@ -0,0 +1,19 @@
+toplevel="$(git rev-parse --show-toplevel)"
+
+# ensure the pre-commit git hook is in place
+pushd $toplevel/.git/hooks >/dev/null &&
+    ln -sf ../../.githooks/pre-commit &&
+    popd >/dev/null
+
+# build the managed go sdk
+pushd $toplevel >/dev/null &&
+    make go-check &&
+    popd >/dev/null
```

Feels a little cleaner to just use a subshell? (pushd and popd are verbose)
```suggestion
(cd "$toplevel"; make go-check)
```

Or, preferably, we can leverage the `-C` option with make:
```suggestion
make -C "$toplevel" go-check
```

---

Yeah - I make extensive use of `git -C` because I manage my home dir as a git repo, but it slipped my mind to do the same with `make`.